.gitignore

```diff
@@ -1,8 +1,9 @@
 .cache/
 bin/
-vendor/
 virtualenv/
 .virtualenv/
 serverthrall.config
 serverthrall.log
 *.pyc
+.idea
+rcedit.exe
```

Comment:
You should put these in your own computers gitignore since they're specific to your editors.